### PR TITLE
New version of the connector

### PR DESCRIPTION
**Pull request by kostadin-JustR**
Lots of new actions added, all previous actions are deprecated and replaced with new. 
All new actions are set to use Businessmap (formerly Kanbanize) latest API v2
Connector re-branded

---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [ ] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

File: certified-connectors/Kanbanize/apiDefinition.swagger.json [hunk 1]
```diff
@@ -1,2706 +1,6145 @@
...
+      "post": {
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {}
+          }
+        },
+        "summary": "Archive Card (deprecated)",
+        "description": "(Deprecated) Move card to archive. The card must be not be blocked and it must be in column Done or in column Archive.",
+        "operationId": "ArchiveCard",
```
Comment (vmanoharas) [inline certified-connectors/Kanbanize/apiDefinition.swagger.json:1986]: Can you please correct the grammar for "must be not be"?

File: certified-connectors/Kanbanize/apiDefinition.swagger.json [hunk 2]
```diff
@@ -1,2706 +1,6145 @@
...
+                  "description": "data"
+                }
+              }
+            }
+          }
+        },
+        "summary": "General Trigger",
+        "operationId": "CardHook",
+        "description": "Triggers for any action on the board. Use Control/Condition or Control/Switch to manage flow for different triggers. Here is partial list of supported event types: \"Card created\", \"Card moved\", \"Card details changed\", \"Card deleted\", \"Card blocked\" \\n\\n Your next step after the trigger should be \"Get Card By ID\" so you can take all card details \\n\\nUse Control/Condition on author ID in bi-drection integrations to prevent loops",
+        "x-ms-trigger": "single",
```
Comment (vmanoharas) [inline certified-connectors/Kanbanize/apiDefinition.swagger.json:2697]: Can you please correct the spelling "drection"

File: certified-connectors/Kanbanize/apiDefinition.swagger.json [hunk 3]
```diff
@@ -1,2706 +1,6145 @@
...
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/getCardTemplatesResponse"
+            }
+          }
+        },
+        "operationId": "GetTemplates",
+		"summary": "Get Templates",
+        "description": "Get all templates witin the account. Used internally.",
```
Comment (vmanoharas) [inline certified-connectors/Kanbanize/apiDefinition.swagger.json]: Can you please correct the spelling "witin"

File: certified-connectors/Kanbanize/Readme.md [hunk 4]
```diff
@@ -1,33 +1,54 @@
 
-## Kanbanize
-An official connector working with the Kanbanize API.
+## Businessmap
+An official connector working with the Businessmap API.
 
 ## Prerequisites
```
Comment (vmanoharas) [inline certified-connectors/Kanbanize/Readme.md]: Could you please refer this template and add required fields in readme.md file?

**Review by vmanoharas — COMMENTED** [review]
Hello @kostadin-kanbanize, 

Can you please resolve review comments?

File: certified-connectors/Kanbanize/apiDefinition.swagger.json (hunk 1 above)
Comment (kostadin-JustR) [reply certified-connectors/Kanbanize/apiDefinition.swagger.json:1986]: fixed

File: certified-connectors/Kanbanize/apiDefinition.swagger.json (hunk 2 above)
Comment (kostadin-JustR) [reply certified-connectors/Kanbanize/apiDefinition.swagger.json:2697]: fixed

File: certified-connectors/Kanbanize/apiDefinition.swagger.json (hunk 3 above)
Comment (kostadin-JustR) [reply certified-connectors/Kanbanize/apiDefinition.swagger.json]: fixed

File: certified-connectors/Kanbanize/Readme.md (hunk 4 above)
Comment (kostadin-JustR) [reply certified-connectors/Kanbanize/Readme.md]: We have added link to our API documentation

**Comment by kostadin-kanbanize** [comment]
Thank you for getting back to us so quick and pointing out these issues. We have corrected them. 
I'm not sure though what you mean by regarding the "required fields" in Readme.md file. Please explain a bit more if we have to add something else in addition to it.

**Comment by vmanoharas** [comment]
Hello @kostadin-kanbanize, 

Please accept my apology I overlooked and did not add the link to refer in my previous message.   Please see the template of readme.md file in the below and make the necessary changes. 

https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md

**Comment by kostadin-kanbanize** [comment]
Hello @vmanoharas, 

We have uploaded updated version of the Readme.md file, which follows the exact template requirements.

**Review by vmanoharas — APPROVED** [review]
Hello @kostadin-kanbanize 
I hope you are doing well.
Congratulations, your pull request is approved and merged. Please [submit the files to ISV Studio](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) or [update your submission](https://learn.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#update-submission-prior-to-certification) using your latest commit ID, please allow us up to 1-2 weeks to review your submission, and our Engineers will notify you in the “Activity Control” area in ISV Studio. Please [make sure that the icon meets all the requirements](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#design-an-icon-for-your-connector), that it’s size is 230x230, icon background is not white or transparent and it matches the iconBrandColor property in apiProperties.json file.
After making the submission in ISV Studio, please attach an intro.md file in the activity control section. This file is required and it's different from readme.md. You can [find an example of intro.md file on this page](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) at step 6.
Please [Create an environment in the Preview region](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#create-an-environment-in-the-preview-region). You will use this environment later to [test your connector](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#test-your-connector) after Microsoft is done with the functional verification for your connector.
We expect all tests to be completed within 1-2 weeks.
If your connector passes all tests, the deployment process begins, and it typically takes up to 3 to 4 weeks to deploy your connector to all of our regions.
Please let me know if you have any questions or run into any problems while creating or updating your submission.
Thank you very much for working with us.

**Pull request merged**